uber/site_sections/schedule.py

```diff
@@ -71,25 +71,15 @@ def schedule_tsv(self, session):
         cherrypy.response.headers['Content-Disposition'] = 'attachment;filename=Schedule-{}.tsv'.format(int(localized_now().timestamp()))
         schedule = defaultdict(list)
         for event in session.query(Event).order_by('start_time').all():
```

Comment:
So the actual fix was just to replace ``.start_time`` with ``.start_time_local`` but I also did some refactoring.

The way this code worked before was that it took ``Event`` instances and then modified them to have extra attributes and to overwrite existing attributes with different data types.  I'm not thrilled with that approach because people might look in the wrong place if they're reading a template and try to look up where something comes from, plus it's dangerous because we might accidentally commit a change we've made to our models.

The safer and more idiomatic approach is to not mutate the object and instead to just make a dict with the values we need, which is what I did here.